src/runtime/runtime.cs

```diff
@@ -339,6 +343,24 @@ internal static void Initialize(bool initSigs = false)
                 loader.Free(dllLocal);
             }
 
+#if MONO_LINUX
```

Comment:
I don't understand why this is here. Code similar to this exists in LibraryLoader.cs, no?

Reply:
Also, in this branch, by the time we get here, the library should already be loaded in memory.

Reply:
It is not loading the python library, but the libraries python may need (libssl, libcrypto, libreadline...), which are in the PythonInstall/lib folder. See the contentn of the [linux binaries archive](https://github.com/Unity-Technologies/com.unity.scripting.python/blob/UT3590/python-binaries-on-linux/support/binaries/linux.7z). 

`loader.Load` is actually the loader from LibraryLoader.cs